relays/bin-substrate/src/cli/mod.rs

```diff
@@ -92,8 +93,9 @@ pub enum Command {
 	ResubmitTransactions(resubmit_transactions::ResubmitTransactions),
 	/// Register parachain.
 	RegisterParachain(register_parachain::RegisterParachain),
-	///
+	/// Relay parachain heads.
```

Comment:
https://github.com/paritytech/parity-bridges-common/commit/97133678203c1688d1aee6801de0c2788eecd691#r125925287

Reply:
(sorry - I've left an original comment for commit, not for the PR)

Reply:
Answered there.